watch-parties/src/main/webapp/index.html

```diff
@@ -17,6 +17,7 @@
         <div id="video-info">
 
             <h2 id="video-title"></h2>
+            <button onclick="setHost()">Host</button>
```

Comment:
Could you add a comment here saying that this is temporary? Maybe a TODO saying it'll be cleaned up once host controls are hooked up to the landing page